Sources/OpenFeature/OpenFeatureAPI.swift

```diff
@@ -4,52 +4,107 @@ import Foundation
 /// A global singleton which holds base configuration for the OpenFeature library.
 /// Configuration here will be shared across all ``Client``s.
 public class OpenFeatureAPI {
-    private var _provider: FeatureProvider? {
-        get {
-            providerSubject.value
-        }
-        set {
-            providerSubject.send(newValue)
-        }
-    }
-    private var _context: EvaluationContext?
+    private let eventHandler = EventHandler()
+    private let queue = DispatchQueue(label: "com.providerDescriptor.queue")
```

Comment:
```suggestion
    private let queue = DispatchQueue(label: "com.openfeature.providerDescriptor.queue")
```

This is totally optional but it may be nice since it looks like the label is only used for debugging purposes.